src/screens/Home/components/ExchangeNotification.tsx

```diff
@@ -57,6 +58,17 @@ export default function ({ data, index }) {
 
   const { fromAmount, toAmount, fromAsset, toAsset } = data.offer;
 
+  React.useEffect(() => {
+    if (timerStatus === STATUS.STARTED) {
+      dispatch(startListeningGetBalancesAction());
+    }
+    if (timerStatus === STATUS.STOPPED) {
+      setTimeout(() => {
+        dispatch(stopListeningGetBalancesAction());
+      }, 1500 * 60);
```

Comment:
Just wondering, do we have any constants for minutes and seconds in milliseconds? If not I think this would be better represented as `90 * 1000` to make it clear that its a 90 second interval

Reply:
Updated it